draft-ietf-jsonpath-base.md

```diff
@@ -1477,8 +1486,8 @@ Notes:
 ### Type Conversion {#type-conv}
 
 Just as queries can be used in logical expressions by testing for the
-existence of at least one node ({{extest}}), a function expression of
-declared type `NodesType` can be used as a function argument for a
+existence of at least one node ({{extest}}), a `nodes-function-expr`
+can be used as a function argument for a
 parameter of declared type `LogicalType`, with the equivalent conversion rule:
```

Comment:
I think it would be better to leave the text unchanged because it is making a semantic rather than a syntactic statement.